IPyth.sol

```diff
@@ -74,6 +74,26 @@ interface IPyth {
     /// @param updateData Array of price update data.
     function updatePriceFeeds(bytes[] memory updateData) external payable;
 
+    /// @notice Wrapper around updatePriceFeeds that rejects fast if a price update is not needed based on the
+    /// sender known latest publish time of prices that it wants to update. This method rejects solely based on
```

Comment:
suggest splitting this first sentence into two: "if a price update is not necessary. A price update is necessary if the publishTime of the update is after the current publishTime on-chain."